test/downstream/integrator_indexing.jl

```diff
@@ -54,8 +54,8 @@ step!(integrator, 100.0, true)
 @test integrator[lorenz1.x] isa Real
 @test integrator[t] isa Real
 @test integrator[α] isa Real
-# @test integrator[γ] isa Real
-# @test integrator[γ] == 2.0
+@test integrator[γ] isa Real
+@test integrator[γ] == 2.0
 
```

Comment:
Test the values and the cases that @YingboMa found broke.

Reply:
no, it doesn't cover the array of symbols handling.

Reply:
So something like `sol[[lorenz1.x, lorenz2.x]]` is required?

Reply:
yes